internal/uvm/create.go

```diff
@@ -70,6 +70,13 @@ type Options struct {
 	// ExternalGuestConnection sets whether the guest RPC connection is performed
 	// internally by the OS platform or externally by this package.
 	ExternalGuestConnection bool
+
+	// DisableCompartmentNamespace sets whether to disable namespacing the network
```

Comment:
Can you add a description here of what it actually means to "namespace the network compartment"? e.g. What behavior does that change?